### ISSUE TITLE
this portion of the fake generator should be unreachable... if you ever see this error, please open an issue

### ISSUE THREAD

**Issue by AlexDShyft** [magidoc-org/magidoc#316]
### Describe the bug

During generation I haven't seen any errors and mostly the docs are working except some pages do not show the examples part (OrderSubmit for example) and throw the error in the console:

Uncaught (in promise) Error: this portion of the fake generator should be unreachable... if you ever see this error, please open an issue: EnrichedBasketEntity
    at cu (FieldDetails.21cf5ce8.js:72:219)
    at Pt (FieldDetails.21cf5ce8.js:66:4207)
    at lu (FieldDetails.21cf5ce8.js:66:3615)
    at FieldDetails.21cf5ce8.js:72:78
    at pages.8456520c.js:53:19097
    at pages.8456520c.js:51:28617
    at at (pages.8456520c.js:51:18776)
    at Function.ny [as mapValues] (pages.8456520c.js:53:19069)
    at FieldDetails.21cf5ce8.js:72:63
    at lu (FieldDetails.21cf5ce8.js:66:3636)

### Reproduction

[schema copy.graphql.txt](https://github.com/magidoc-org/magidoc/files/14015542/schema.copy.graphql.txt)

### System Info

```shell
System:
    OS: macOS 14.2.1
    CPU: (10) arm64 Apple M2 Pro
    Memory: 155.16 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.17.1 - ~/.nvm/versions/node/v18.17.1/bin/node
    npm: 9.6.7 - ~/.nvm/versions/node/v18.17.1/bin/npm
  Browsers:
    Chrome: 120.0.6099.234
    Safari: 17.2.1
```

### Severity

Serious, but I can work around it

**Comment by pelletier197** [comment]
Hey! Sorry for the late reply, I did see your message when it came out but totally forgot to get back to you. Sorry about that. 

I have just released 4.1.3, and it seems to me like your schema file is working on my side. Would you be able to give it another try with the latest version? Thank you

**Comment by pelletier197** [comment]
@AlexDShyft I'll close this as it has been inactive for almost 2 months. Feel free to re-open if you still face the issue.
